enterprise-suite/templates/kube-state-metrics-deployment.yaml

```diff
@@ -45,6 +45,7 @@ spec:
           image: {{ .Values.kubeStateMetricsImage }}:{{ .Values.kubeStateMetricsVersion }}
           args:
             - --port=8080
+            - --namespace={{ .Values.kubeStateMetricsScrapeNamespaces }}
```

Comment:
Don't we want a conditional check here, so that if nothing is specified we don't set this option at all? Or is the blank value equivalent to wildcard?

Reply:
We don't need the condition check.   Based on the official docs, if we don't set it, it will use the default value "".  We set this value to default value in values.yaml already

Here is the official doc
https://github.com/kubernetes/kube-state-metrics/blob/master/docs/cli-arguments.md